gulp/get-build-info.js

```diff
@@ -0,0 +1,45 @@
+const git = require('git-last-commit');
+const { promisify } = require('util');
+
+// Holds cached commit info, so we don't need to
+// re-read it every time.
+let commitInfo;
```

Comment:
I started to always set a variable to null instead of leaving it undefined. It makes the starting state explicit: `let commitInfo = null;`